lib/presentation/workout/workout_page.dart

```diff
@@ -44,14 +43,7 @@ class WorkoutPage extends StatelessWidget {
           child: Text("This workout doesn't have any exercises! Would you like to add some?"),
         );
       } else {
-        return ListView(
```

Comment:
Why was this removed?

Reply:
It was missing an argument and there wasn't an appropriate one to supply so i decided to remove it for now since we werent using the page atm either